src/Jenssegers/Mongodb/Relations/MorphTo.php

```diff
@@ -53,4 +54,16 @@ protected function whereInMethod(EloquentModel $model, $key)
     {
         return 'whereIn';
     }
+
+    /**
+     * @inheritdoc
```

Comment:
i suggest to change or remove this doc block

Reply:
Done.